app-backend/api/src/main/scala/project/Routes.scala

```diff
@@ -160,6 +164,14 @@ trait ProjectRoutes extends Authentication
     }
   }
 
+  def acceptScenes(projectId: UUID): Route = authenticate { user =>
+    entity(as[List[UUID]]) { sceneIds =>
+      onSuccess(Future.sequence(sceneIds.map(s => ScenesToProjects.acceptScene(projectId, s)))) { _ =>
```

Comment:
I think we need to handle failure in a friendlier fashion. This isn't a great error message: `[A]List[A]: MoveRight,MoveRight,MoveRight,MoveRight,MoveRight,MoveRight,DownArray`.